lib/rails/auth/x509/middleware.rb

```diff
@@ -11,22 +11,15 @@ module X509
       class Middleware
         # Create a new X.509 Middleware object
         #
-        # @param [Object]               app next app in the Rack middleware chain
-        # @param [String]               ca_file path to the CA bundle to verify client certs with
-        # @param [Hash]                 cert_filters maps Rack environment names to cert extractors
-        # @param [Logger]               logger place to log verification successes & failures
-        # @param [Boolean]              require_cert causes middleware to raise if certs are unverified
-        # @param [OpenSSL::X509::Store] truststore (optional) provide your own truststore (for e.g. CRLs)
+        # @param [Object] app next app in the Rack middleware chain
```

Comment:
I can't seem to add my review directly on line 9, but that comment is no longer true. It might be worth adding a bit of context that you've added to the PR description.

Reply:
Similarly, I believe `CertificateVerifyFailed` (line 7) is unused now.